thunderfish/csvmaker.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from .dataloader import load_data
+from .consistentfishes import consistent_fishes
+from .harmonicgroups import harmonic_groups
+from .powerspectrum import multi_resolution_psd
+
+
+def extract_main_freqs_and_db(data, samplerate):
+    psd_data = multi_resolution_psd(data, samplerate, fresolution=[0.5, 2 * 0.5, 4 * 0.5])
+
+    # find the fishes in the different powerspectra and extract fund. frequency and power
+    fishlists = []
+    for i in range(len(psd_data)):
+        fishlist = harmonic_groups(psd_data[i][1], psd_data[i][0])[0]
+        fishlists.append(fishlist)
+
+    filtered_fishlist = consistent_fishes(fishlists)
+    tmp = np.vstack([filtered_fishlist[e][0] for e in range(len(filtered_fishlist))])
+
+    freqs = tmp[:, 0]
+    pows = tmp[:, 1]
+    decibel = 10.0 * np.log10(pows)  # calculate decibel using 1 as P0
+    return freqs, decibel
+
+
+def freqs_and_db_csv(filename, freq, decibel):
+    header = ['fundamental frequency', 'dB']
```

Comment:
Actually, I just checked out numpy's savetext function:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.savetxt.html#numpy.savetxt
It already can do everything we want. You may just write a really simple function that calls np. savetext with the appropriate parameters for writing a csv file.